stdlib/public/Darwin/Accelerate/vDSP_FillClearGenerate.swift

```diff
@@ -0,0 +1,762 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import Accelerate
+
+/// Types that support vectorized window generation.
+@available(iOS 9999, OSX 9999, tvOS 9999, watchOS 9999, *)
+public protocol vDSP_FloatingPointGeneratable: BinaryFloatingPoint {
+}
+extension Float: vDSP_FloatingPointGeneratable {}
+extension Double: vDSP_FloatingPointGeneratable {}
+
+extension vDSP {
+    
+    /// Fill vector with specified scalar value, single-precision.
+    ///
+    /// - Parameter vector: The vector to fill.
+    /// - Parameter value: The fill value.
+    @inline(__always)
+    @available(iOS 9999, OSX 9999, tvOS 9999, watchOS 9999, *)
+    public static func fill<V>(_ vector: inout V,
+                               with value: Float)
+        where V: _MutableContiguousCollection,
+        V.Element == Float {
+            
+            let n = vDSP_Length(vector.count)
+            
+            vector.withUnsafeMutableBufferPointer { v in
+                withUnsafePointer(to: value) {
+                    vDSP_vfill($0,
+                               v.baseAddress!, 1,
+                               n)
+                }
+            }
+    }
+    
+    /// Fill vector with specified scalar value, double-precision.
+    ///
+    /// - Parameter vector: The vector to fill.
+    /// - Parameter value: The fill value.
+    @inline(__always)
+    @available(iOS 9999, OSX 9999, tvOS 9999, watchOS 9999, *)
+    public static func fill<V>(_ vector: inout V,
+                               with value: Double)
+        where V: _MutableContiguousCollection,
+        V.Element == Double {
+            
+            let n = vDSP_Length(vector.count)
+            
+            vector.withUnsafeMutableBufferPointer { v in
+                withUnsafePointer(to: value) {
+                    vDSP_vfillD($0,
+                                v.baseAddress!, 1,
+                                n)
+                }
+            }
+    }
+    
+    /// Fill vector with zeros, single-precision.
+    ///
+    /// - Parameter vector: The vector to fill.
+    /// - Parameter value: The fill value.
```

Comment:
Comment references a parameter that does not exist.

Reply:
Fixed for both single- and double-precision clear functions.